### PR TITLE
refactor(explorer): move explorer routes into its own route group

### PR DESCRIPTION
**Pull request by holic**
I wanna use a new route group for some internal stuff that shouldn't inherit the base layout.tsx and apparently need to move things into route groups to "opt out" of a layout.

### CODE REVIEW

**Comment by holic** [comment]
I had some concerns about using import path aliases to avoid all the `../../../..` when refactoring/moving files in VSCode, but sounds like some of those issues might be addressed in upcoming TS 5.6: https://github.com/microsoft/TypeScript/issues/59119

**Review by karooolis — APPROVED** [review]

**Comment by holic** [comment]
needs more work for cases when you load this up with no world address

gonna defer this for now

File: packages/explorer/src/middleware.ts [hunk 1]
```diff
@@ -1,16 +0,0 @@
-import { NextRequest, NextResponse } from "next/server";
-
-export function middleware(request: NextRequest) {
-  const pathname = request.nextUrl.pathname;
-  const worldAddress = process.env.WORLD_ADDRESS;
-
-  if ((pathname === "/" || pathname === "/worlds") && worldAddress) {
```
[the remark this answers is not in the document]
Comment (holic) [reply packages/explorer/src/middleware.ts:7]: this and it helps render the 404 page for routes within explorer

[the remark this answers is not in the document]
Comment (karooolis) [reply packages/explorer/src/middleware.ts:7]: Which ones should valid routes moving forward, and which ones 404?

At the moment it's:
- `/` -> `/worlds`
- `/worlds` -> `/worlds/${worldAddress}`
- Everything else 404s ?

**Pull request merged**